queens-psyc-clinic/app
# FE Sign-in/ Sign-up Pages

### **Sign-in/out Pages**
This is the `/sign-in`   `/sign-up`   `/account-type` pages done! 

_Note: Still have to fix input restrictions to catch if correct formatting of emails, names etc!_ 

### **Testing**
1. navigate to branch: `git checkout signup-signin`
2. `cd` to the `public` folder
3. run `npm install`
4. run `npm start`
5. ensure the pages are working (yes I know the inputs aren't working right - fix will come later)
<br>

For reference checkout issue: #43
